test/service/test_request_validation.py

```diff
@@ -249,19 +270,111 @@ def test(url, message, params):
                     test(url,
                          params=dict(catalog=self.catalog,
                                      some_nonexistent_filter=1),
-                         message='Invalid query parameter `some_nonexistent_filter`')
+                         message="Invalid parameter 'some_nonexistent_filter'")
                 with self.subTest(test='malformed parameter'):
                     test(url,
                          params=dict(catalog=self.catalog,
                                      size='foo'),
-                         message='Invalid value for parameter `size`')
+                         message="Invalid value for parameter 'size'")
                 with self.subTest(test='malformed filter parameter'):
                     test(url,
                          params=dict(catalog=self.catalog,
                                      filters='{"}'),
-                         message='The `filters` parameter is not valid JSON')
-        url = self.base_url + '/integrations'
-        with self.subTest(test='missing required parameter'):
-            test(url,
-                 params={},
-                 message='Missing required query parameters `entity_type`, `integration_type`')
+                         message="The 'filters' parameter is not valid JSON")
+        if "/integrations" in self.app_module.app.specs['paths'].keys():
```

Comment:
Nitpick:
```suggestion
        if '/integrations' in self.app_module.app.specs['paths'].keys():
```

Reply:
not a nit, its a requirement from contribution guidelines ha.